src/Auth0.php

```diff
@@ -245,6 +253,11 @@ public function __construct(array $config)
             throw new CoreException('Invalid domain');
         }
 
+        $this->discoveryDocumentUri = $config['discovery_document_uri'] ?? $_ENV['AUTH0_DISCOVERY_DOCUMENT_URI'] ?? null;
```

Comment:
Why are we asking for this if we already have the `domain`? The discovery document has a spec-defined URL, `.well-known/openid-configuration`

Reply:
You are right, I've forced openid-configuration